### ISSUE TITLE
[windows] unable to delete an already deleted git repository 

### ISSUE THREAD

**Issue by juzam** [keybase/client#8835]
Steps to reproduce:

- add a keybase repo as a `new-origin` remote
- launch the command `git push --all new-origin` and interrupt it (CTRL-C works well, but could also be caused by a connectivity problem)
- try to delete said keybase repository

The first deletion apparently doesn't give an error but the result is a spurious entry in the list of repositories, that can't be removed. 
On the next tries to delete it it just fails with the error `repository xxx doesn't exist` so the entry still exsists in the list.

**Comment by maxtaco** [comment]
thanks for the report. cc: @oconnor663

**Comment by oconnor663** [comment]
Good catch! We'll need to make sure we're getting "doesn't exist" as a concrete error type from KBFS, but either way it should be a quick fix.
